modules/ROOT/pages/component/remove.adoc

```diff
@@ -0,0 +1,109 @@
+= Remove a component
+:description: This section describes how to remove components from Neo4j Ops Manager.
+
+[IMPORTANT]
+====
+Metrics related to a component that has been removed (either directly or indirectly) are removed as part of the scheduled daily clean-up process for metrics.
+They are not visible once the component has been removed but may remain in the persistence storage until the clean-up process has run.
+====
+
+
+[[agent]]
+== Agent
+
+=== Conditions
+Only agents that have never contacted the server or where the `agent cleanup safe-period` has been exceeded since last contact, can be removed.
+
+[NOTE]
+====
+`agent cleanup safe-period` can be set with the *topology.agent.cleanup.safe-period* application property.
+
+The default value is *PT1H*.
+====
+
+=== Trigger location
+
+Agents listing in Global Settings
+
+image::agents-remove-hover.png[width=800]
+
+=== Related components that will be removed
+
+* *All* Instances monitored by this agent.
+* *All* Database instances belonging to instances monitored by this agent.
+* *Any* databases that will no longer have any database instances.
+* *Any* DBMSs that will no longer contain any instances.
+
+
+[[instance]]
+== Instance
+
+=== Conditions
+It is only possible to remove instances that have been offline for longer than the 'instance cleanup safe-period'
+
+[NOTE]
+====
+`instance cleanup safe-period` can be set with the *topology.instance.cleanup.safe-period* application property.
+
+The default value is *PT1H*.
+====
+
+=== Trigger location
+. Context menu for instance node in Status panel
+
+image::status-panel-remove-instance.png[width=800]
+
+
+=== Related components that will be removed
+
+* *All* Database instances belonging to this instance.
+* *Any* databases that will no longer have any database instances.
+* *Any* DBMSs that will no longer contain any instances.
+
+[[dbms]]
+== DBMS
+
+=== Conditions
+It is only possible to remove DBMSs if all of its instances have been offline for longer than the `instance cleanup safe-period`.
```

Comment:
```suggestion
It is only possible to remove a DBMS if all of its instances have been offline for longer than the `instance cleanup safe-period`.
```
alternatively:
It is only possible to remove DBMSs if all of their instances have been offline for longer than the `instance cleanup safe-period`.